server/controller/root.py

```diff
@@ -2,27 +2,28 @@
 Root Controller
 """
 import functools
-import controller
 
 from bottle import abort, post, redirect, request, response, route, static_file
 from kajiki_view import kajiki_view
 from markupsafe import Markup
 
+from helpers.config_manager import ConfigManager
```

Comment:
You can't. Just tried it and it fails since ConfigManager is not a package (it's a class). I did change it to alias ConfigManager => config